pandas/core/series.py

```diff
@@ -2423,25 +2423,26 @@ def unstack(self, level=-1, fill_value=None):
 
     def map(self, arg, na_action=None):
         """
-        Map values of Series using input correspondence (which can be
-        a dict, Series, or function)
+        Map values of Series using input correspondence (a dict, Series, or
+        function).
 
         Parameters
         ----------
         arg : function, dict, or Series
+            Mapping correspondence.
```

Comment:
I think we should add here some details on how a dict and Series are handled.

Reply:
This is outside the original scope of the contribution, see first PR comment.